### PR TITLE
DIGI_14_CreateTaxParameter

### PR DESCRIPTION
**Pull request by prakashkolli**
1.created TaxParameter DataModel To insert into database
2.Created Controller

### CODE REVIEW

File: DigiOrderApi/DigiOrderApi/Controllers/TaxParameterController.cs [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        {
+            try
+            {
+                taxParameterDataModel = new TaxParameterDataModel();
+                dtRecords = taxParameterDataModel.PostTaxParameter(data);
+                dtRecords.TableName = "TAXPARAMETERS";
+            }
+            catch (Exception ex)
+            {
+                return Request.CreateErrorResponse(HttpStatusCode.BadRequest, "Please Enter Proper Data");
```
Comment (anebhargavkrishna) [inline DigiOrderApi/DigiOrderApi/Controllers/TaxParameterController.cs:39]: Change to internal server error

File: DigiOrderApi/DigiOrderApiDataAccess/TaxParameterDataModel.cs [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+{
+    public class TaxParameterDataModel
+    {
+        DataTable dtResult = null;
+
+        //Author: Prakash
+        //6-03-2020
+        //Description: Insert into a Tax Parameter to Database
+        //Reviewer
+        //Review Version:01
```
Comment (anebhargavkrishna) [inline DigiOrderApi/DigiOrderApiDataAccess/TaxParameterDataModel.cs:16]: Remove comments

File: DigiOrderApi/DigiOrderApi/Controllers/TaxParameterController.cs (hunk 1 above)
Comment (prakashkolli) [reply DigiOrderApi/DigiOrderApi/Controllers/TaxParameterController.cs]: changed

File: DigiOrderApi/DigiOrderApiDataAccess/TaxParameterDataModel.cs (hunk 2 above)
Comment (prakashkolli) [reply DigiOrderApi/DigiOrderApiDataAccess/TaxParameterDataModel.cs]: removed comments as per your comments

**Pull request merged**